internal/federation/handle.go

```diff
@@ -45,6 +45,7 @@ func MakeJoinRequestsHandler(s *Server, w http.ResponseWriter, req *http.Request
 		Type:       "m.room.member",
 		StateKey:   &userID,
 		PrevEvents: []string{room.Timeline[len(room.Timeline)-1].EventID()},
+		Depth:      room.Timeline[len(room.Timeline)-1].Depth() + 1,
```

Comment:
This is a drive-by fix and doesn't affect the outcome of the test.

Reply:
Please don't mix this with tests where possible. The set of reviewers for $feature are different from the set of reviewers for handling Complement internals.